packages/react-core/src/components/MenuToggle/MenuToggle.tsx

```diff
@@ -57,31 +64,65 @@ export class MenuToggleBase extends React.Component<MenuToggleProps> {
     const isPlain = variant === 'plain';
     const isPlainText = variant === 'plainText';
 
-    const content = (
+    const toggleControls = (
```

Comment:
`toggleControls` is used also [here](https://github.com/patternfly/patternfly-react/pull/7748/files/7eebfbf42cbbea9aa30c8aac2e8b522139400d6d#diff-3a010e5b45739c99d841592b45e81bcf5f852ce5ffb6a22f58f7665d1664e033R117), I refactored the original function so I was able to avoid code duplication

Reply:
Ah gocha, I missed that! Good thinking on that